### PR TITLE
Implement basic tab navigation

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-1397

### CODE REVIEW

File: packages/frontend/src/scripts/configureTableIndexRerender.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* 
+  If you want to rerender indexes of a table (e.g. when you filter rows and want to re-render indexes),
+  you need to add the following attributes to the table element:
+  - data-role="table"
+  - data-table-index-rerender="true"
```
Comment (antooni) [inline packages/frontend/src/scripts/configureTableIndexRerender.ts:5]: shouldn't it be table-indexes-renderer-on ?

**Review by antooni — APPROVED** [review]
kudos

File: packages/frontend/src/scripts/configureTableIndexRerender.ts (hunk 1 above)
Comment (torztomasz) [inline packages/frontend/src/scripts/configureTableIndexRerender.ts:5]:
```suggestion
  - data-table-indexes-renderer-on="true"
```

File: packages/frontend/src/scripts/configureTableIndexRerender.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  You also need to add the following attributes to the index cell:
+  - data-role="index-cell"
+
+  Example:
+  <table data-role="table" data-table-index-rerender="true">
```
Comment (torztomasz) [inline packages/frontend/src/scripts/configureTableIndexRerender.ts:11]:
```suggestion
  <table data-role="table" data-table-indexes-renderer-on="combined-bridges-checkbox">
```

**Pull request merged**